dependencies/graknlabs/dependencies.bzl

```diff
@@ -44,26 +44,26 @@ def graknlabs_grakn_core():
     git_repository(
         name = "graknlabs_grakn_core",
         remote = "https://github.com/graknlabs/grakn",
-        commit = "391912c48bbe2a140f78cc4c2bef67226155e8d6", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
+        commit = "49fc93ad98442b57533b64e22775b564f25e4a43", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
     )
 
 def graknlabs_protocol():
     git_repository(
         name = "graknlabs_protocol",
-        remote = "https://github.com/graknlabs/protocol",
-        tag = "1.0.4", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_protocol
+        remote = "https://github.com/jmsfltchr/protocol",
+        commit = "87c5b205ab7b8747e05e9310cc8e64bdc34c41bc", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_protocol
     )
 
 def graknlabs_verification():
     git_repository(
         name = "graknlabs_verification",
-        remote = "https://github.com/graknlabs/verification",
+        remote = "git@github.com:graknlabs/verification.git",
```

Comment:
Are we keeping this using ssh just for private repos now?

Reply:
Good question, a question for the team

Reply:
I didn't know you can provide the SSH address here in `dependencies.bzl`. @vmax is this right? it seems to work, but is this an expected pattern in Bazel?

Reply:
@haikalpribadi you can, and you actually should — otherwise cloning fails with a cryptic error

Reply:
you mean that we should always use `git@` for all `git_repository()` ? We should make this sweeping change in all repositories then! I didn't know. @vmax

Reply:
by "always", I mean for both `public` and `private` repos? or is it just meant for `private` repos?

Reply:
@haikalpribadi nope, I only meant those that are not public, such as `@graknlabs_verification`

Reply:
what happens if you use `git@` for all `git_repository()` @vmax ?

Reply:
@haikalpribadi then we probably won't be able to clone them in CI unless we specifically add the SSH key in CircleCI settings